book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] This is a close one! but the `rentals` attribute in the book class should be able to be modified from outside the class so I'd suggest you use the `attr_accessor` instead.